src/components/PageWithSideMenu.tsx

```diff
@@ -76,22 +78,38 @@ interface LinkButtonProps extends ButtonProps, Partial<Pick<LinkProps, 'to'>> {
 
 export const LinkButton = ({ isSelected, sx, ...rest }: LinkButtonProps) => (
   <Button
-    sx={{ bgcolor: isSelected ? 'primary.main' : 'background.default', ml: '1rem', width: 'fit-content', ...sx }}
+    sx={{ bgcolor: isSelected ? 'primary.main' : 'background.default', justifyContent: 'start', ...sx }}
     variant={isSelected ? 'contained' : 'outlined'}
     LinkComponent={rest.to ? Link : undefined}
     {...rest}
   />
 );
 
 interface LinkIconButtonProps extends LinkButtonProps {
-  icon: ReactNode;
+  selectedColor?: string;
 }
 
-export const LinkIconButton = ({ sx, icon, ...rest }: LinkIconButtonProps) => (
-  <LinkButton sx={{ ml: '0rem', mr: '1rem', ...sx }} {...rest}>
-    {icon}
-  </LinkButton>
-);
+export const LinkIconButton = ({ sx, title, isSelected, selectedColor, ...rest }: LinkIconButtonProps) => {
```

Comment:
Dette er kanskje ikke bare en `IconButton` lenger?
```suggestion
export const LinkCreateButton = ({ sx, title, isSelected, selectedColor, ...rest }: LinkCreateButtonProps) => {
```

Reply:
`LinkCreateButton` er også et dårlig navn, men kommer ikke på no bra i farta..

Reply:
Eneste jeg kan tenke på som alternativ er "LinkNewButton", men vil ikke si at det er noe bedre.